src/daft-session/src/python.rs

```diff
@@ -1,14 +1,23 @@
+use std::sync::Arc;
+
 use daft_catalog::{
     python::{PyCatalogWrapper, PyIdentifier, PyTable, PyTableSource, PyTableWrapper},
     Identifier,
 };
+use daft_dsl::functions::python::WrappedUDFClass;
 use pyo3::prelude::*;
 
 use crate::Session;
 
 #[pyclass]
 pub struct PySession(Session);
 
+impl PySession {
+    pub fn inner(&self) -> &Session {
+        &self.0
+    }
+}
```

Comment:
If we want it public, then named is better, but either or works here 

```suggestion
#[pyclass]
pub struct PySession {
     pub session: Session // alt. 1
}

impl PySession {
    pub fn session(&self) -> &Session {
        &self.0 // alt. 2
    }
}
```